lib/internal/abort_controller.js

```diff
@@ -197,7 +197,10 @@ class AbortSignal extends EventTarget {
   static any(signals) {
     validateAbortSignalArray(signals, 'signals');
     const resultSignal = createAbortSignal({ composite: true });
-    const resultSignalWeakRef = new WeakRef(resultSignal);
+    let resultSignalWeakRef;
+    if (signals.length) {
```

Comment:
This is not what I meant. What I meant is to put the whole `for` in an if. and then just use const inside.

Reply:
yes, that's the same